tools/transformer/src/transformations/oneOf.js

```diff
@@ -135,15 +136,28 @@ function handleDuplicates(parentObject, childObject) {
   ]);
   if (duplicates.length > 0) {
     const duplicatesSource = childObject.title || "";
-    const missmatches = duplicates.filter((e) => e.typeMismatch);
+    let missmatches = duplicates.filter((e) => e.typeRefMismatch);
```

Comment:
very nit (non-blocking): curious about the different spelling missmatch/mismatch - would be nice if we stayed consistent

Reply:
is actually missmatch an allowed word? Maybe I am just ignoring it as a non-native speaker

Reply:
Nice find. `Mismatch` is correct spelling.

Reply:
Resolved